lms/djangoapps/notes/migrations/0002_auto_20190709_2011.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.22 on 2019-07-09 20:11
+from __future__ import unicode_literals
```

Comment:
We will want to hold off on this deletion until the next PR: https://openedx.atlassian.net/wiki/spaces/AC/pages/23003228/Everything+About+Database+Migrations#EverythingAboutDatabaseMigrations-Deploymentandbackward-compatiblemigrations